index.html

```diff
@@ -12,21 +11,15 @@
     </div>
 </div>
 
-
-
-
 <div class="img-container">
     <img style="width:100%;" src="{{ site.baseurl }}/assets/img/home/so-helpful-cropped-all3.jpg"/>
-    <div class="bottom-left"><h1>We are a group of University of Manitoba students who support women in technology.</h1></div>
+    <div class="bottom-left"><h1>We are a group of University of Manitoba students who support women in technology. All are welcome to join!</h1></div>
```

Comment:
I inspected the site and I'm not sure why this div is here... Unless someone else can find a purpose, can you just delete it?